client/client-benchmark-noop-api-plugin/src/main/java/org/opensearch/plugin/noop/action/bulk/RestNoopBulkAction.java

```diff
@@ -38,8 +38,8 @@
 import org.opensearch.action.bulk.BulkShardRequest;
 import org.opensearch.action.support.ActiveShardCount;
 import org.opensearch.action.update.UpdateResponse;
-import org.opensearch.client.Requests;
-import org.opensearch.client.node.NodeClient;
+import org.opensearch.client.server.Requests;
```

Comment:
To me (just an opinion), `org.opensearch.client.server` is somewhat obscure package name, couple of alternatives: 
- `org.opensearch.internal.client.*
- `org.opensearch.action.client.*` 
- `org.opensearch.transport.client.*`
- `org.opensearch.support.client.*`

@prudhvigodithi @andrross what do you think folks?

Reply:
I'm ok with `org.opensearch.internal.client`.

Reply:
I like `internal.client`.

I would also have accepted `server.client` (not the other way around), because it's the server's client, not the client's server.

Reply:
Thanks I have refactored the code with `org.opensearch.internal.client`, please check.

Reply:
I hate to be that guy, but I have a couple problems with "internal":
- these classes (particularly `org.opensearch.client.Client`) are one of the primary interfaces used in the public plugin API.
- "internal" is not really that descriptive. There could be an "internal" REST client, for example. And with the move towards opensearch-java as the public Java client, the rest client implementation inside this repo does in fact become "internal".

I vote for `org.opensearch.transport.client.*` because this is the transport client, which is what differentiates it from the other `org.opensearch.client` namespace. @msfroh @reta what do you think?

Reply:
`org.opensearch.node.client.*` might work too, as this is the client for node-to-node communication. The biggest issue is that I would prefer that downstream repos should not need to have imports in the `org.opensearch.internal` namespace, as the obvious implication is that such classes are intended to be internal to their repo.

Reply:
`org.opensearch.transport.client.*` looks good to me as well

Reply:
Today we have `transport` a package inside `package org.opensearch.client.transport;` which has just `NoNodeAvailableException`. we should be able to move this outside and continue with `org.opensearch.transport.client`.

Reply:
I'm also onboard with `org.opensearch.transport.client` :+1: